sdk/python/kfp/_client.py

```diff
@@ -304,6 +312,31 @@ def create_experiment(self, name, description=None, namespace=None):
       IPython.display.display(IPython.display.HTML(html))
     return experiment
 
+  def get_pipeline_id(self, name, page_size=100):
+    """Returns the pipeline id if a pipeline with the name exsists.
+    Args:
+      name: pipeline name
+      page_token: token for starting of the page.
+      page_size: size of the page.
```

Comment:
reminder, also the docstring

Reply:
sorry, multi tasking while listening in to some meetings.